gulpfile.js

```diff
@@ -191,8 +193,11 @@ gulp.task('clobber-tests', function() {
  * assets.
  */
 gulp.task('offline', ['build'], function() {
-  gulp.src(['**/*'], { cwd: DIST_APP_ROOT })
-    .pipe(gsww())
+  gulp.src(['**/*'], {cwd: DIST_APP_ROOT})
+    .pipe(gsww({
+      version: package.version,
```

Comment:
We use it to add it to the serviceworker as a variable.

With that, with each new app version our sw code is different, making the browser to update the SW code, caches, etc.